### PR TITLE
Make donation box on mobile smaller

### PR DESCRIPTION
**Pull request by epicfaace**
test copy

### CODE REVIEW

File: wp-content/themes/thestanforddaily/sass/site/_site.scss [hunk 1]
```diff
@@ -88,6 +88,9 @@ $header_donation_banner_close_button_offset: 20px;
 		.tsd-donation-form {
 			margin-top: 5px;
 		}
+		* {
+			font-size: 12px;
```
Comment (hesyifei) [inline wp-content/themes/thestanforddaily/sass/site/_site.scss]: Why is this needed?

Comment (epicfaace) [reply wp-content/themes/thestanforddaily/sass/site/_site.scss]: Otherwise I would need to add selectors for the button, select, and other elements in the donate header. ¯\_(ツ)_/¯
